recipes/cassandra/activate.sh

```diff
@@ -0,0 +1,3 @@
+
+export OMP_NUM_THREADS=4
```

Comment:
This is a bad idea and not user friendly. At minimum, this script should backup any existing values. TBH, I would remove the activate script all together and make users responsible.

Reply:
Thanks for the feedback! For this code, OMP_NUM_THREADS=4 is a *very reasonable* choice in most cases. My goal is to give the user the best possible starting point. How about storing the original value in another environment variable and restoring it upon deactivation? I am open to other suggestions as well, but I don't love the idea of leaving the user completely responsible.

Reply:
>  For this code, OMP_NUM_THREADS=4 is a very reasonable choice in most cases.

It should still be managed by the user and not the package, you don't know what other packages will be installed in the same environment. I regularly using machines where 4 threads would cause the processed to be killed and others where I want to use at least 10x more.

If this really should be more carefully managed for special reasons that should be modified in the package itself.

Reply:
Ok, thanks for the input. I will remove the activate/deactivate scripts.

Reply:
This setting is very problematic. For example, this will force any library with OpenMP in it on the machine in this env to use 4 threads. This will include other packages beyond cassandra in the environment. At minimum, you need to have a cassandra specific thread env var if you want to do this. 

I urge you to reconsider your approach here. Ultimately, the user's environment is their responsibility, not yours. While this may seem to help, in my experience at the end of the day, having environmental stuff like this too packaged up leads to inflexible packages. Another way to think about what I am saying is that we are shipping tools to people, not fully fledged computing environments. Our job so to speak is to package up the pieces in a way that makes them easy to install and interoperable. After that, there is not much we can or should be doing. Setting this env var breaks the interoperability with other packages and so is a bad idea for sure.

Reply:
jynx @chrisburr !

Reply:
@beckermr @chrisburr 

Thanks again for the feedback!